app/models/plug/resources/site_notice.rb

```diff
@@ -10,6 +10,10 @@ def enabled?
       def disabled?
         state == 'disabled'
       end
+
+      def icon
```

Comment:
This could potentially be a concern or in a helper since it looks like you are using the same method in 2 classes. Maybe something to think about if you end up needing it in a third :)

Reply:
Right. @yarhtut use `Themable` concern name.